fbpic/particles/gathering/cuda_methods.py

```diff
@@ -346,23 +346,23 @@ def gather_envelope_field_gpu_linear(x, y, z,
                         exptheta_m, a_m,
                         iz_lower, iz_upper, ir_lower, ir_upper,
                         S_ll, S_lu, S_lg, S_ul, S_uu, S_ug,
-                        flip_factor=minus_one_m )
```

Comment:
It seems that gpu kernels do not support passing arguments by keywords (i.e. with `=`).